packages/synapse-react-client/src/components/ChallengeSubmission/AvailableEvaluationQueueList.tsx

```diff
@@ -0,0 +1,153 @@
+import { HelpOutlineTwoTone } from '@mui/icons-material'
+import {
+  Autocomplete,
+  Box,
+  List,
+  ListItem,
+  MenuItem,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material'
+import { Evaluation } from '@sage-bionetworks/synapse-types'
+import { noop } from 'lodash-es'
+import React, { useState } from 'react'
+
+type TextWithHelpIconProps = {
+  text: string
+  tooltipTitle?: string
+}
+
+function TextWithHelpIcon(props: TextWithHelpIconProps) {
+  const { text, tooltipTitle } = props
+  return (
+    <>
+      {text}
+      {tooltipTitle && (
+        <Tooltip title={tooltipTitle}>
+          <HelpOutlineTwoTone
+            sx={{
+              ml: 1,
+              color: 'grey.600',
+              fontSize: '16px',
+            }}
+          />
+        </Tooltip>
+      )}
+    </>
+  )
+}
+
+type AvailableEvaluationQueueStaticListProps = Pick<
+  AvailableEvaluationQueueListProps,
+  'evaluations'
+>
+
+function AvailableEvaluationQueueStaticList(
+  props: AvailableEvaluationQueueStaticListProps,
+) {
+  const { evaluations } = props
+
+  return (
+    <Box mt={2}>
+      <Typography variant="body1">Available Evaluation Queues:</Typography>
+      <List dense={true}>
+        {evaluations.map(evaluation => {
+          return (
+            <ListItem key={evaluation.id}>
+              <TextWithHelpIcon
+                text={evaluation.name!}
+                tooltipTitle={evaluation.submissionInstructionsMessage}
+              />
+            </ListItem>
+          )
+        })}
+      </List>
+    </Box>
+  )
+}
+
+type AvailableEvaluationQueueAutocompleteListProps = Pick<
+  AvailableEvaluationQueueListProps,
+  'evaluations' | 'onChangeSelectedEvaluation'
+>
+
+function AvailableEvaluationQueueAutocompleteList(
+  props: AvailableEvaluationQueueAutocompleteListProps,
+) {
+  const { evaluations, onChangeSelectedEvaluation = noop } = props
+  const [value, setValue] = useState<Evaluation | null>(null)
+
+  return (
+    <Autocomplete
+      // Must be set to false for dropdown list to be correctly positioned
+      // when component is in a modal opened from a Markdown Widget in SWC
+      disablePortal={false}
```

Comment:
In SWC, when the Submission modal is opened from a markdown widget, the dropdown list is not positioned correctly. Setting `disablePortal` to `false` so that the Popper content will not be under the DOM hierarchy of the parent component prevents this issue.